services/headless-lms/server/src/controllers/auth.rs

```diff
@@ -263,12 +363,13 @@ pub fn _add_routes(cfg: &mut ServiceConfig) {
         .burst_size(10)
         .finish()
         .unwrap();
-    cfg.service(
-        web::resource("/login")
-            .wrap(Governor::new(&governor_conf))
-            .to(login),
-    )
-    .route("/logout", web::post().to(logout))
-    .route("/logged-in", web::get().to(logged_in))
-    .route("/authorize", web::post().to(authorize_action_on_resource));
+    cfg.route("/signup", web::post().to(signup))
```

Comment:
* Add a new rate limit to this with a new governor_conf, so that the signup gets its own rate limit.
* Add the secret http header to tmc requests so that the secret project does not get rate limited